R/testdata.R

```diff
@@ -903,10 +903,10 @@ dataset_test_worker <-
             info = paste0(red(f), "\ttaxonomic_update")
           )
           taxon_names <- sapply(metadata[["taxonomic_updates"]], "[[", "find")
-          expect_is_in(
-            unique(taxon_names), data[[metadata[["dataset"]][["taxon_name"]]]] %>% unique(),
-            info = paste0(red(f), "\ttaxonomic_updates"), label = "`taxon_name`'s"
-          )
+          # expect_is_in(
```

Comment:
Please add a comment in the file saying why it's commented out and what needs to be done. Otherwise we'll come back to this and wonder "Why is this commented out?"